app/routes/admin.py

```
@@ -11033,6 +10847,94 @@ def passkey_settings():
 
     # ==================== ISSUE RESOLUTION SYSTEM ====================
 
+
+def _resolve_issue_identity(actor_public_id, class_public_id):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_resolve_issue_identity`

(ANN202)